### DISCUSSION TITLE
TuYa SS9600ZB not recognized

### DISCUSSION THREAD

**Discussion by VanJ2** [Koenkk/zigbee2mqtt#15790] [Devices]
Hi all. 
I have paired this device ([https://www.zigbee2mqtt.io/devices/SS9600ZB.html#tuya-ss9600zb](https://www.zigbee2mqtt.io/devices/SS9600ZB.html#tuya-ss9600zb)) to network, so I thought it would be recognized and supported. However it is not. The Z2M page displays it like this:
![image](https://user-images.githubusercontent.com/119114713/209854416-e1e5da31-d7be-45ba-aaf2-d42d7b2f82cc.png)
Am I doing something wrong?

**Comment by zhizaichenggong** [comment]
邮件已收到，谢谢

**Comment by rodrigofragadf** [comment]
Same problem here.
